hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/JobCoordinationService.java

```diff
@@ -400,6 +400,18 @@ void completeSnapshot(long jobId, long executionId, long snapshotId, boolean isS
         }
     }
 
+    boolean shouldStartJobs() {
```

Comment:
You can remove the `shouldStartJobs` from `restartJob` method in this class.

Reply:
good spotting

Reply:
your comment is addressed.